x-pack/plugins/observability/public/pages/alert_details/components/alert_summary.tsx

```diff
@@ -41,6 +39,8 @@ export interface AlertSummaryProps {
   alertSummaryFields?: AlertSummaryField[];
 }
 
+const DEFAULT_DATE_FORMAT = 'MMM D, YYYY @ HH:mm:ss.SSS';
```

Comment:
I know this is not part of the refactoring, but we should keep a not to use uiSettings and the user's defined dateFormat instead of this hard coded format.

Reply:
I agree. There are multiple definitions of this const in 2 or 3 pages. Seems like they can be centralized in a better way.